### ISSUE TITLE
Lab2: Part2_Debiasing Test Set Bias

### ISSUE THREAD

**Issue by rvh72** [aamini/introtodeeplearning#110]
Hello,
I have a question on test set bias of the standard CNN model. After running cell #13 in Part2_Debiasing.ipynb, I see this histogram:

<img width="916" alt="Lab2_Cell13_Std_CNN_Test_Set_Bias" src="https://user-images.githubusercontent.com/85761183/163728355-39fb70ae-c820-415e-ade5-cc619b0e5986.png">

Am I correct in interpreting that avg test set accuracy is ~65% for Light Female, ~70% for Light Male, ~82% for Dark Female, and ~90% for Dark Male?

If training set has majority of data on light skinned female, I expected test set accuracy to be higher for that category. So, results of above hisogram are counter intuitive.

What am I missing?

--Rahul

**Comment by markusjohnsson** [comment]
Yeah I'm getting unexpected results too, did you find an explanation @rvh72 ? Interestingly I'm not getting the same distribution as you, but I'm still seeing lowest probability (?) for Light Female.

**Comment by rvh72** [comment]
No, haven't found any explanation. This sort of a base result does not indicate any need for de-biasing the dataset. What distribution did you get?
